src/Avalonia.Controls/WindowBase.cs

```diff
@@ -126,6 +126,16 @@ public void Activate()
             PlatformImpl?.Activate();
         }
 
+        public void EnableCloseButton()
```

Comment:
I'm not sure these methods should be added to `WindowBase`, shouldn't they be on `Window`? Popups don't have a close button.